python/hail/experimental/ldscore.py

```diff
@@ -188,12 +188,17 @@ def ld_score(entry_expr,
     ht_scores = ht_scores.key_by('idx')
     ht_scores = ht_scores.rename({'f{:}'.format(i): col_idxs[i]
                                   for i in range(len(cols))})
+    ht_scores._force_count()
 
     ht = mt.select_rows(__locus=locus_expr).rows()
     ht = ht.add_index()
-    ht = ht.annotate(**ht_scores[ht.idx])
+    ht = ht.annotate(**ht_scores[ht.idx]) # dies here
```

Comment:
what dies here?

Reply:
(the other comment that I appear to have forgotten to save---the `force_count()`s should probably be removed?)

Reply:
Wow, I completely failed at cleaning up from a long debugging session.